Basic-Video-Chat-SPM/README.md

```diff
@@ -0,0 +1,13 @@
+Basic Video Chat sample app
+===========================
+
+The Basic Video Chat app is a very simple application meant to get a new developer
+started using the VonageVideo iOS SDK. 
+
+Swift Package Manager
+---------------------
+In this example the VonageVideo iOS SDK was already included as a Swift Package Dependency.
```

Comment:
@IGitGotIt, good point.

@jvalli, can we just update an existing sample app, since SPM ships instead, like Jaideep suggested?  It's good that we did this to prove that it works end-to-end, though. 💪

Reply:
I tested this on the existing swift sample app only. I agree to @IGitGotIt point totally